configs/swin_transformer_v2/metafile.yml

```diff
@@ -16,7 +16,7 @@ Collections:
     README: configs/swin_transformer_v2/README.md
 
 Models:
-  - Name: swinv2-tiny_16xb64_in1k.py
+  - Name: swinv2-tiny-w8_16xb64_in1k-256px
```

Comment:
The `Name` in metafile should be the same as the checkpoint name instead of the config file. For example, If we trained a model by ourselves and we want to provide both checkpoints.

Reply:
got it, fixed